packages/stores/src/ui-config/errors.ts

```diff
@@ -22,6 +22,17 @@ export class InvalidSwapFeeError extends Error {
   }
 }
 
+export class InvalidScalingFactorControllerAddress extends Error {
+  constructor(m: string) {
+    super(m);
+    // Set the prototype explicitly.
+    Object.setPrototypeOf(
+      this,
+      InvalidScalingFactorControllerAddress.prototype
+    );
```

Comment:
Question) Can I know the reason setting prototype explicitly?

Reply:
This pattern was adopted from what tony did in keplr-wallet. I'm not sure of why it is needed, to be honest.

Reply:
Oh, It's for extending Error class.
https://stackoverflow.com/questions/41102060/typescript-extending-error-class